doc/source/user_guide/dsintro.rst

```diff
@@ -712,11 +712,6 @@ The ufunc is applied to the underlying array in a :class:`Series`.
    ser = pd.Series([1, 2, 3, 4])
    np.exp(ser)
 
-.. versionchanged:: 0.25.0
-
-   When multiple :class:`Series` are passed to a ufunc, they are aligned before
```

Comment:
It might make sense to keep the comments, just without the version changed directive.

Reply:
Got it, so we have to keep the comments, just remove the directives right?

Reply:
Yep, also have to remove the tab before the comment. Has to be on the same level as the comments before and after